tools/serverpod_cli/test/generator/dart/client_code_generator/class_copy_with_test.dart

```diff
@@ -138,8 +137,12 @@ void main() {
           expect(copyWithMethod?.returnType?.toSource(), testClassName);
         }, skip: copyWithMethod == null);
 
+        test('annotated with @useResult', () {
```

Comment:
I see that this file also should be migrated to the current pattern, but I deemed this out of scope for now to get the quick-win in and focus on the other TODOs next.

Reply:
It does actually have the description style, but perhaps a little too segmented in different layers.

Reply:
Yeah, I just thought we would avoid `group` for what is really a (dependent) test, and `test` for individual assertions.

But happy to refactor this some other time, then we can clarify any open question in there :)